sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/oauth2/AadOAuth2AuthenticatedPrincipal.java

```diff
@@ -1,192 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-package com.azure.spring.cloud.autoconfigure.aad.implementation.oauth2;
-
-import com.azure.spring.cloud.autoconfigure.aad.AadResourceServerWebSecurityConfigurerAdapter;
-import com.nimbusds.jwt.JWTClaimsSet;
-import com.nimbusds.jwt.JWTClaimsSet.Builder;
-import org.springframework.security.core.GrantedAuthority;
-import org.springframework.security.oauth2.core.OAuth2AuthenticatedPrincipal;
-import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationConverter;
-import org.springframework.util.Assert;
-
-import java.io.Serializable;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Map;
-import java.util.Map.Entry;
-
-import static org.springframework.security.core.authority.AuthorityUtils.NO_AUTHORITIES;
-
-/**
- * Entity class of AADOAuth2AuthenticatedPrincipal
- *
- * @deprecated use the default converter {@link JwtAuthenticationConverter} instead in {@link AadResourceServerWebSecurityConfigurerAdapter}.
- */
-@Deprecated
-public class AadOAuth2AuthenticatedPrincipal implements OAuth2AuthenticatedPrincipal, Serializable {
```

Comment:
Is there a replacement for this?

Reply:
Will use the default converter `JwtAuthenticationConverter` to return `JwtAuthenticationToken` instead, this deprecation was marked in this PR https://github.com/Azure/azure-sdk-for-java/pull/29471